tabpy-tools/tabpy_tools/rest.py

```diff
@@ -322,18 +329,17 @@ def to_json(self):
 
     def __eq__(self, other):
         return (
-            type(self) == type(self)
-            and all((
-                getattr(self, a) == getattr(other, a)
-                for a in self.__rest__
-            )))
+                type(self) == type(self)
```

Comment:
This looks like it should be `type(self) == type(other)` though I wonder if you even need to override this method in the first place?

Reply:
Good eye! Thank you for catching this!

We need to implement the methods for the type to be used as a dictionary key.